data/language/en-GB.txt

```diff
@@ -4296,7 +4296,7 @@ STR_DTLS    :Hired by a large amusement park chain, your task is to build them a
 <Gravity Gardens>
 STR_SCNR    :Gravity Gardens
 STR_PARK    :Gravity Gardens
-STR_DTLS    :Your challenge is to build a roller coaster park in the beautiful Gravity Gardens - No other rides, just roller coasters!
+STR_DTLS    :Your challenge is to build a roller coaster park in the beautiful Gravity Gardens - no other rides, just roller coasters!
```

Comment:
This one has exclamation points, which makes for a unique instance.  I would say all the scenario descriptions should be consistent, with periods if any have punctuation at all.  In this particular instance and the others below I think it's more appropriate to replace the - with a comma.